src/emulator/gui/src/settings_dialog.cpp

```diff
@@ -92,13 +93,38 @@ using namespace list;
 void draw_settings_dialog(HostState &host) {
     ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
     ImGui::Begin("Settings", &host.gui.configuration_menu.settings_dialog, ImGuiWindowFlags_AlwaysAutoResize);
-    ImGuiTabBarFlags settings_tab_flags = ImGuiTabBarFlags_None;
+    const auto settings_tab_flags = ImGuiTabBarFlags_None;
     ImGui::BeginTabBar("SettingsTabBar", settings_tab_flags);
 
-    // Core
-    if (ImGui::BeginTabItem("Core")) {
+    // General
+    if (ImGui::BeginTabItem("General")) {
         ImGui::PopStyleColor();
-        if (fs::exists(host.pref_path + "vs0/sys/external")) {
+        if (ImGui::Button("Reset Configuration")) {
+            host.cfg = Config{};
+
+            LOG_INFO("Reset Vita3K configuration and config file to default values.");
+            if (host.cfg.pref_path != host.pref_path) {
+                if (change_pref_location(host.cfg.pref_path, host.pref_path)) {
+                    host.pref_path = host.cfg.pref_path;
+
+                    // Refresh the working paths
+                    app::serialize_config(host.cfg, host.cfg.config_path);
+                    LOG_INFO_IF(app::clear_and_refresh_game_list(host), "Successfully moved Vita3K files to: {}", host.pref_path);
+                }
+            }
+        }
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Reset Vita3K's configuration to the default values. \nAfter pressing, restart Vita3K to fully apply. \nWARNING: This cannot be undone.");
+        ImGui::EndTabItem();
+    } else {
+        ImGui::PopStyleColor();
+    }
+
+    // Modules
```

Comment:
don't change this, keep Core, i have set this for can add in futuru Other backend cpu